api/pkg/apis/v1alpha1/providers/target/staging/staging.go

```diff
@@ -100,7 +100,8 @@ func (i *StagingTargetProvider) Get(ctx context.Context, deployment model.Deploy
 		i.Context.SiteInfo.CurrentSite.BaseUrl,
 		deployment.Instance.Name+"-"+i.Config.TargetName,
 		i.Context.SiteInfo.CurrentSite.Username,
-		i.Context.SiteInfo.CurrentSite.Password)
+		i.Context.SiteInfo.CurrentSite.Password,
+		scope)
```

Comment:
I not totally sure but this seems to assume that catalog need to be stored in the same namespace as instance scope. Is that appropriate?

Reply:
No, it's not appropriate. Good catch! I am reverting it "default".